client/components/Item/UpNext.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <v-fade-transition>
+    <v-card class="container white--text pa-4">
+      <div class="d-flex flex-column flex-grow-1">
+        <v-card-title class="text-h6 pa-0 my-1 mx-0">
+          {{ $t('dialog.upNext.nextItemPlayingIn') }}
```

Comment:
I'm not sure this is properly translated either.

Different languages have different orders for words. So while English may be "The next item is playing in 15 seconds", another language might be equivalent to "15 seconds until the next item".

A good way to circumvent this while still using a separate "units" translation is to chain translations:
```typescript
$t('dialog.upNext.nextItemPlayingIn', { seconds: $tc('units.time.seconds', { value: timeLeft }))
```

There might be a nicer way to do this, but this avoids duplicating a translation for units.